tests/test_browsers.py

```diff
@@ -78,10 +105,30 @@ def test_chrome_linux(become_linux, change_homedir):  # noqa: F811
             "www.github.com",
         ),
     )
-    profs = f.profiles(f.history_file)
-    his_path = f.history_path_profile(profs[0])
+    assert_bookmarks_equal(
+        bmk[0],
+        (
+            datetime.datetime(
+                2021,
+                2,
+                9,
+                4,
+                7,
+                32,
+                tzinfo=datetime.timezone(datetime.timedelta(seconds=19800), "IST"),
```

Comment:
I realized for cases where the profiles stored for testing use different timezones thes tests fail for instance here you have used "IST" and whoever wrote the tests on histories used "CEST". I don't know how this might be resolved @sadielbartholomew you seem to be the expert on timezones.

Reply:
Yuup thats the error! But doesnt the detach timezone stuff supposed to negate whatever time zone the histories or bookmarks have?

Reply:
Its a bit weird tbh, UTC and IST have a time difference of 5:30 hrs yet the day, month , year and time is the name